pom.xml

```diff
@@ -9,11 +9,11 @@
     </parent>
     <groupId>io.jenkins.plugins</groupId>
     <artifactId>signpath</artifactId>
-    <version>${revision}${changelist}</version>
+    <version>${changelist}</version>
     <packaging>hpi</packaging>
     <properties>
         <revision>2.0.1</revision>
-        <changelist></changelist>
+        <changelist>999999-SNAPSHOT</changelist>
         <jenkins.version>2.359</jenkins.version>
         <java.level>11</java.level>
```

Comment:
these shouldn't be overridden unless you really know what you are doing. 
line 18-22 I think could probably be deleted

Reply:
I pushed one more commit here 
https://github.com/jenkinsci/signpath-plugin/pull/25/commits/ba9ad3d04e7af9b694c02c803bf049de40e6cb0f

The goal is to have a manually controlled version prefix and auto-increment for the build number.